x/reward/keeper/claim_reward.go

```diff
@@ -3,18 +3,15 @@ package keeper
 import (
 	"bytes"
 	"encoding/binary"
-	"log"
 	"math/big"
 
 	errorsmod "cosmossdk.io/errors"
 	sdkmath "cosmossdk.io/math"
 	"github.com/ExocoreNetwork/exocore/x/assets/types"
 	rtypes "github.com/ExocoreNetwork/exocore/x/reward/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
-	"github.com/evmos/evmos/v16/rpc/namespaces/ethereum/eth/filters"
```

Comment:
removing this import would trigger very weird test failures, in case anybody might meet similar problems, I would describe the cause and how to fix it: both evmos repo and exocore repo have some init functions inside app module that would try to initialize the account address prefix, when we import or indirectly import the app package, the init function would automatically run to set the prefix as `evmos` or `exo`. Sometimes if we remove some import, it could cause the order of executing init functions changes, thus resulting the expected prefix `exo` being set as `evmos`. Of course we can force to set prefix as `exo` for every keeper test suite, but a very accidental case would still cause unexpected failures: In some unit tests that would not be initialized by baseTestSuite, if we happened to call the `String` method on some `AccAddress` acc, the bech32 representation of this address would be cached. Since we don't force setting prefix as `exo` in unit tests, these bech32 representations could be `evmos.....`. And if we happen to use the same address in our keeper test suite, the cached bech32 representation would be returned when we want to get such address's string representation, and even cause failure when we try to recover bytes representation from the bech32 string, since the prefix is incorrect